tark/tark/templates/documentation.html

```diff
@@ -58,148 +135,3640 @@ <h4 id="tark_we">What services are provided by Web Tark?</h4>
+													</code></pre>
+												</div>
+											</div>	
+
+											<div id="menu_api_translation_translation_id_wget" class="tab-pane fade"  style='width: 90%;'>
+												<div class='tab-pane' id='api_translation_translation_id_wget'>
+													<pre class="pre-scrollable prettyprint linenums"><code class="wget">
+wget -q --header='Content-type:application/json' 'http://tark.ensembl.org/api/translation/1/'  -O -
+													</code></pre>
+												</div>
+											</div>	
+
+
+											</div>											
+										</div>											
+										<!-----Example request - END -------------------------->											
+									</div>
+								</div>                
+							</tbody>
+						</table> 
+					</div>    
+					<hr/>   
+					<!-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+					---------------------------------------------------------------------FAQ Q8 -------------------------------------------------------------------------------------------
+					--------------------------------------------------------------------------------------------------------------------------------------------------------------------------->    
+					<h4 id="ar_tech_stack">What is the technology stack of Tark?</h4>
+					<p><a href="https://www.python.org/">Python3</a>, <a href="https://www.djangoproject.com/">Django</a>, <a href="http://www.django-rest-framework.org/">Django REST Framework</a> and <a href="https://django-rest-swagger.readthedocs.io/en/latest/">Django REST Swagger</a>. <br>Backend support is provided by<a href="https://www.mysql.com/"> MySQL</a>. </p>
+    
+					<a href="#tark_summary">Go Top</a>
+				
+				</div> <!-- closing div tag for div class="panel-body" id="tark_faq" name="tark_faq" -->
+			</div> <!-- closing div tag for  div class="panel panel-info" -->   
+		</div> <!-- closing div tag for class="col-md-9" -->   
+	</div> <!-- closing div tag for div class="row" -->
+</div> <!-- closing div tag for div class="container" -->
+
+<script>
+<!---------------------------------------------------------------------------------->
+<!------------------------Add Javascript for collapsible------------------>
+<!---------------------------------------------------------------------------------->
+var coll = document.getElementsByClassName("collapsible");
+var i;
+for (i = 0; i < coll.length; i++) {
+	coll[i].addEventListener("click", function() {
+		this.classList.toggle("active-expand");
+		var content = this.nextElementSibling;
+		if (content.style.maxHeight){
+			content.style.maxHeight = null;
+		} else {
+			content.style.maxHeight = content.scrollHeight + "px";
+		} 
+	});
+}
+
+<!---------------------------------------------------------------------------------->
+<!-----------------------------Add Javascript for modal-------------------->
```

Comment:
It's best practice to have JavaScript in a separate file (e.g. a file inside `static/js` directory)